### DISCUSSION TITLE
Github classrooms: autograding tests uneditable

### DISCUSSION THREAD

**Discussion by pohjus** [community/community#88516] [GitHub Education]
### Hi everyone,

I am creating autograding tests in github classrooms and I am able to create the tests but I **_cannot edit the tests._** The edit button **does not do anything**. My old tests that I did last week are something that I am able to edit. But new ones are uneditable.

I have reproduced this with:

- **macOS Sonoma 14.2.1**: Chrome (120.0.6099.199), Safari (7.2.1) and MS Edge (120.0.2210.133)
- **Windows 10**: Chrome (120.0.6099.224)

Here is the gif animation of the result. Any ideas how to fix this issue?

![2024-01-17 10 30 10](https://github.com/community/community/assets/5622604/39e939ae-eae0-42f6-bbe9-d99807fe573d)

**Comment by gpincheiraa** [comment]
Same problem here!
